freud/order/RotationalAutocorrelation.h

```diff
@@ -41,7 +41,7 @@ std::pair<std::complex<float>, std::complex<float>> quat_to_greek(const quat<flo
  *  SO(3). This computation can be performed using a hyperspherical coordinate
  *  representation of the rotations. For details, see "Design rules for
  *  engineering colloidal plastic crystals of hard polyhedra – phase behavior
- *  and directional entropic forces" by Karas et al. (currently in preparation).
+ *  and directional entropic forces" :cite:`Karas2019`.
```

Comment:
```suggestion
 *  and directional entropic forces": https://doi.org/10.1039/C8SM02643B.
```
`:cite:` is a Sphinx command, it does not mean anything in a C++ file.